### PR TITLE
added reply_to param in assign/remind/revoke emails.

### PR DESCRIPTION
**Pull request by zamanafzal**
## Description

Describe what this pull request changes, and why these changes were made. How will these changes affect other people, installations of edx, etc.?
Please include links to any relevant ADRs, design artifacts, and decision documents. Make sure to document the rationale behind significant changes in the repo, per [OEP-19](https://open-edx-proposals.readthedocs.io/en/latest/oep-0019-bp-developer-documentation.html), and can be
linked here.

Useful information to include:
- Which edX user roles will this change impact? Common user roles are "Learner", "Course Author", "Developer", and "Operator".
- Include screenshots for changes to the UI (ideally, both "before" and "after" screenshots, if applicable).
- Provide links to the description of corresponding configuration changes. Remember to correctly annotate these changes.

## Supporting information

Link to other information about the change, such as Jira issues, GitHub issues, or Discourse discussions.
Be sure to check they are publicly readable, or if not, repeat the information here.

## Testing instructions

Please provide detailed step-by-step instructions for testing this change; how did YOU test this change?

## Other information

Include anything else that will help reviewers and consumers understand the change.
- Does this change depend on other changes elsewhere?
- Any special concerns or limitations? For example: deprecations, migrations, OpenEdx vs. edx.org differences, development vs. production environment differences, security, or accessibility.

### CODE REVIEW

File: ecommerce/enterprise/tests/test_utils.py [hunk 1]
```diff
@@ -388,6 +389,19 @@ def test_get_enterprise_customer_sender_alias(self, sender_alias, expected_sende
         sender_alias = get_enterprise_customer_sender_alias('some-site', 'uuid')
         assert sender_alias == expected_sender_alias
 
+    @patch('ecommerce.enterprise.utils.get_enterprise_customer')
+    @ddt.data(
+        ('edx@example.com', 'edx@example.com'),
```
Comment (irfanuddinahmad) [inline ecommerce/enterprise/tests/test_utils.py:394]: Do we need ddt here?

File: ecommerce/enterprise/tests/test_utils.py [hunk 2]
```diff
@@ -388,6 +389,19 @@ def test_get_enterprise_customer_sender_alias(self, sender_alias, expected_sende
...
+    )
+    @ddt.unpack
+    def test_get_enterprise_customer_reply_to_email(self, reply_to, expected_reply_to, enterprise_customer):
+        """
+        Verify get_enterprise_customer_reply_to_email returns enterprise reply_to email/
+        """
+        enterprise_customer.return_value = {'reply_to': reply_to}
+        reply_to = get_enterprise_customer_reply_to_email('some-site', 'uuid')
+        assert reply_to == expected_reply_to
```
Comment (irfanuddinahmad) [inline ecommerce/enterprise/tests/test_utils.py]: Always use assertEqual(), as it customizes failure output.

File: ecommerce/enterprise/utils.py [hunk 3]
```diff
@@ -675,3 +675,28 @@ def get_enterprise_customer_sender_alias(site, enterprise_customer_uuid):
...
+    Returns:
+        reply_to: email if the enterprise_customer has a reply to email address, empty string otherwise.
+    """
+    reply_to = ''
+    try:
+        enterprise_customer = get_enterprise_customer(site, enterprise_customer_uuid)
+        reply_to = enterprise_customer['reply_to']
+    except Exception as exc:  # pylint: disable=broad-except
+        logging.exception(
+            '[Enterprise Reply to Email Fetch Failure]. Customer: %s, Site: %s, Exception: %s',
```
Comment (irfanuddinahmad) [inline ecommerce/enterprise/utils.py]: "Enterprise Reply to Email Fetch Failure" -> "Enterprise - failure in fetching reply_to property"

**Review by irfanuddinahmad — COMMENTED** [review]
Some minor suggestions

**Review by saleem-latif — APPROVED** [review]

**Review by irfanuddinahmad — APPROVED** [review]
LGTM!

**Pull request merged**